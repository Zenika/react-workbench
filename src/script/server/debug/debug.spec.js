/* eslint-env jest */
const debug = require('./debug')

describe('services/debug', () => {
  describe('get', () => {
    it('should return debug informations', () => {
      expect(debug.get()(undefined, () => ({ some: 'redux informations' }))).toMatchSnapshot()
    })
  })
})
