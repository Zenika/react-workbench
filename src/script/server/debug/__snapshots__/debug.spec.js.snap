// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`services/debug get should return debug informations 1`] = `
Object {
  "redux": Object {
    "some": "redux informations",
  },
}
`;
